sci-biology/salmon/salmon-0.6.0.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
```

Comment:
Sure, my comments from yesterday have gone because the PR has been updated but I had made the same remark to aim for EAPI 6 and that's how the context came about.